assets/src/components/reader-theme-selection/theme-card.js

```diff
@@ -13,31 +14,31 @@ import PropTypes from 'prop-types';
 /**
  * Internal dependencies
  */
-import './style.css';
-import { __ } from '@wordpress/i18n';
-import { Options } from '../../components/options-context-provider';
-import { Selectable } from '../../components/selectable';
-import { Phone } from '../../components/phone';
+import { Options } from '../options-context-provider';
+import { Selectable } from '../selectable';
+import { Phone } from '../phone';
+import { AMPNotice, NOTICE_TYPE_WARNING, NOTICE_SIZE_SMALL } from '../amp-notice';
 
 /**
  * A selectable card showing a theme in a list of themes.
  *
  * @param {Object} props Component props.
  * @param {string} props.description Theme description.
+ * @param {string} props.currentlyActiveThemeNotice The notice to show if the theme is unselectable.
  * @param {string} props.homepage Link to view more information about the theme.
  * @param {string} props.screenshotUrl URL for screenshot of theme.
  * @param {string} props.slug Theme slug.
  * @param {string} props.name Theme name.
- * @param {boolean} props.unavailable Whether the theme is not automatically installable in the current environment.
+ * @param {boolean} props.disabled Whether the theme is not automatically installable in the current environment.
  */
-export function ThemeCard( { description, homepage, screenshotUrl, slug, name, unavailable } ) {
+export function ThemeCard( { currentlyActiveThemeNotice, description, homepage, screenshotUrl, slug, name, disabled } ) {
 	const { editedOptions, updateOptions } = useContext( Options );
 	const { reader_theme: readerTheme } = editedOptions;
 
 	const id = `theme-card__${ slug }`;
 
 	return (
-		<Selectable className={ `theme-card` } direction="bottom" HTMLElement="li" selected={ readerTheme === slug }>
+		<Selectable className={ `theme-card` } direction="bottom" ElementName="li" selected={ readerTheme === slug }>
```

Comment:
The capital letter indicates that it's going to be used as a JSX component. This is a convention that some ESLint configs enforce so it's a habit, although technically it's not required. I'll go ahead and make it small because it doesn't matter in this code base.

Reply:
Never mind. `<elementName>` did end up breaking it. No wonder I had that habit 😸